src/main/java/org/sonatype/repository/conan/internal/proxy/ConanProxyFacet.java

```diff
@@ -139,8 +147,8 @@ private Content putMetadata(final String assetPath,
       TempBlob blob = tempBlob;
       switch (assetKind) {
         case DOWNLOAD_URL:
-          blob = absoluteUrlRemover.removeAbsoluteUrls(tempBlob, getRepository());
-          // break intentionally left out!
+          String assetName = group + "/" + project + "/" + version + "/index.json";
```

Comment:
Minor: The variable name "assetName" isn't clear here, and makes it sound like it will be the name of download_urls. Maybe call this indexAssetName. Also, Why do we build this here, it seems like it is an internal implementation detail of absoluteUrlRemover?

Reply:
I built it here because I believe I should be able to reuse the urlRemover fort the root indexing which will require the use of different naming

Reply:
Okay then I'd improve the variable name.